history-of-the-quakers-v3/original/13-book-4-chapter-27.adoc

```diff
@@ -74,14 +77,16 @@ Afterwards they sailed to the coast of Barbary,
 where the mate undertook to put them ashore in their own country;
 and after some time of consideration how it might be done in safety, he took them,
 with the help of some of the men, in the boat,
-so near the more that they might wade to it, which at hs desire they readily did;
+so near the more that they might wade to it, which at his desire they readily did;
 and because they said they were four miles from any town,
 he gave them some bread and other necessaries.
 Then putting the boat closer in, they threw them their arms on shore,
 and put off to return to their ship.
 Thus manifesting in deeds their obedience to the
 precepts of our Saviour--Love your enemies,
-and do good to them that hate you.], was master,
+and do good to them that hate you.]
+
```

Comment:
This new line should not have been introduced, because it causes a paragraph break, where there wan't one before.  it's kind of confusing since it's a long footnote in the middle of a sentence, which is unusual.  Just letting you know in case you run across one like this again.  I'll delete the extra line before I merge 👍